thinc/backends/_custom_kernels.py

```diff
@@ -55,7 +51,7 @@
     cupy.RawModule(
         code=KERNELS_SRC, options=("--std=c++11",), name_expressions=KERNELS_LIST
     )
-    if cupy is not None
+    if has_cupy_gpu
```

Comment:
`has_cupy` or `has_cupy_gpu` (I am not sure)? I think the kernels are only compiled at `__call__` time.

Reply:
We need `has_cupy_gpu` here as `_get_kernel` gets called during module init, which in turn calls into `cupy`. The latter call will raise an exception if there are no visible GPU devices (even if `cupy` is installed, i.e, `has_cupy` returns `True`).